servers/store/src/main/java/org/projectnessie/server/store/TableCommitMetaStoreWorker.java

```diff
@@ -246,7 +247,7 @@ public Content applyId(Content content, String id) {
   }
 
   @Override
-  public Content.Type getType(ByteString onRefContent) {
+  public Content.Type getType(byte payload, ByteString onRefContent) {
     ObjectTypes.Content parsed = parse(onRefContent);
```

Comment:
Can we determine type based on `payload` now?

Reply:
**Should** work - but probably not guaranteed for old values.
The follow-up PRs will actually handle the `payload` value and deal with the case when `payload==0`.